crates/bdk/tests/psbt.rs

```diff
@@ -99,8 +99,8 @@ fn test_psbt_fee_rate_with_witness_utxo() {
     assert!(finalized);
 
     let finalized_fee_rate = psbt.fee_rate().unwrap();
-    assert!(finalized_fee_rate.as_sat_per_vb() >= expected_fee_rate);
-    assert!(finalized_fee_rate.as_sat_per_vb() < unfinalized_fee_rate.as_sat_per_vb());
+    assert!(finalized_fee_rate.to_sat_per_vb_ceil() as f32 >= expected_fee_rate);
+    assert!(finalized_fee_rate.to_sat_per_vb_ceil() <= unfinalized_fee_rate.to_sat_per_vb_ceil());
 }
```

Comment:
If we provided a method for creating a `FeeRate` from a float with something like 
```rust
fn from_sat_per_vbf(sat_vb: f64) -> FeeRate {...} 
```

then the above can be simplified

```rust
let expected_fee_rate = FeeRate::from_sat_per_vbf(1.2345);
builder.fee_rate(expected_fee_rate);
...
assert!(finalized_fee_rate >= expected_fee_rate);
assert!(finalized_fee_rate < unfinalized_fee_rate);
```